tests/acceptance/characters/test_freerider_attacks.py

```diff
@@ -46,12 +47,8 @@ def test_policy_simple_sinpa(blockchain_ursulas, blockchain_alice, blockchain_bo
     for ursula in blockchain_ursulas:
         # Reset the Ursula for the next test.
         ursula.suspicious_activities_witnessed['freeriders'] = []
-        try:
-            with ursula.datastore.query_by(PolicyArrangement, writeable=True) as arrangements:
-                [arrangement.delete() for arrangement in arrangements]
-        except RecordNotFound:
-            # No records were found; this Ursula didn't have the arrangement.
-            continue
+        with find_policy_arrangements(ursula.datastore, writeable=True) as arrangements:
+            [arrangement.delete() for arrangement in arrangements]
```

Comment:
I know this was not introduced in this PR, but I find this to be an add pattern, since it's producing a list of `None`s.  This is an abuse of list comprehensions IMO.